content/swiftui/concepts/viewmodifier/terms/fontWeight/fontWeight.md

```diff
@@ -0,0 +1,56 @@
+---
+Title: '.fontWeight()'
+Description: 'Sets the font weight of the text in Text view.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Modifiers'
+  - 'Xcode'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.fontWeight()`** modifier sets the font weight. We can apply the following weights:
```

Comment:
We should move the sentence "We can apply the following font weights:" and the following bulleted list to the Syntax section after the ```pseudo code block.